concepts/classes/introduction.md

```diff
@@ -0,0 +1,95 @@
+# Introduction
+
+It is time to get to one of the core principles of C++: object-oriented programming (OOP).
```

Comment:
"principles" -> "paradigms" (at least that's what it's most often called in my experience)